lib/cpl.rb

```diff
@@ -186,6 +189,18 @@ def self.all_base_commands
     rescue StandardError => e
       ::Shell.abort("Unable to load command: #{e.message}")
     end
+
+    private
+
+    def puts_info_header(config)
+      rows = {}
+      rows["ORG"] = config.org unless config.org.nil? || config.org.empty?
+      rows["APP"] = config.app unless config.app.nil? || config.app.empty?
+
+      rows.each do |key, value|
+        puts "#{key}: #{value}"
+      end
+    end
```

Comment:
I think we should put a newline at the end to separate the header from the rest of the command output.